package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "tensorboard",
+  "version": "0.0.0-unused",
+  "description": "TensorFlow's visualization toolkit",
+  "private": true,
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "postinstall": "ngc -p angular-metadata.tsconfig.json",
+    "build": "bazel build //...",
+    "test": "ibazel test //..."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tensorflow/tensorboard.git"
+  },
+  "keywords": [
+    "ml",
+    "visualization"
+  ],
+  "author": "TensorFlow authors",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/tensorflow/tensorboard/issues"
+  },
+  "homepage": "https://github.com/tensorflow/tensorboard#readme",
+  "devDependencies": {
+    "@angular/bazel": "^8.1.2",
+    "@angular/cli": "^8.1.2",
+    "@angular/compiler": "^8.1.2",
+    "@angular/compiler-cli": "^8.1.2",
+    "@bazel/bazel": "^0.23.2",
```

Comment:
This PR bumps the Bazel version in `WORKSPACE` from 0.22.0 to 0.26.1,
but adds a dependency in `package.json` at version 0.23.2. Is there any
reason for the discrepancy?

Cf. <https://github.com/tensorflow/tensorboard/issues/2534#issuecomment-520073054>